src/domains/index.root.ts

```diff
@@ -0,0 +1,16 @@
+export class DomainRoot {
+  /** For now it is private */
+  private intoUniqueArray<T>(arr: T[]): T[] {
```

Comment:
```suggestion
  private intoUniqueArray(arr: string[]): string[] {
```
to string? not the T?

Reply:
handled